pyproject.toml

```diff
@@ -39,6 +40,12 @@ lightkube = "^0.15.2"
 psycopg2 = "^2.9.9"
 spark8t = { git = "https://github.com/canonical/spark-k8s-toolkit-py.git", tag="v0.0.7", python=">3.8,<4.0"}
 
+[tool.poetry.group.charm-libs.dependencies]
+ops = "^2.13.0"
+poetry-core = "^1.9.0"
+# grafana_agent/v0/cos_agent.py requires pydantic <2
+pydantic = "^1.10.17, <2"
```

Comment:
**praise** Nice! I really believe we should add these acrsso the repositories @theoctober19th @welpaolo